bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestFilters.java

```diff
@@ -384,14 +386,36 @@ public void testRowFilterBinaryComparator() throws Exception {
     Assert.assertArrayEquals(rowKey1, results[0].getRow());
     Assert.assertArrayEquals(rowKey3, results[1].getRow());
 
-    // Test BinaryComparator - NO_OP
-    filter = new RowFilter(CompareFilter.CompareOp.NO_OP, rowKey2Comparable);
-    results = scanWithFilter(table, rowKey1, rowKey4, qual, filter);
-    Assert.assertEquals("# results", 0, results.length);
-
     table.close();
   }
 
+  @Test
+  public void testEqualsRowFilterBinaryComparator() throws Exception {
```

Comment:
Please use this naming: testRowFilterBinaryComparatorXXXXX or testRowFilterBinaryComparator_XXXXX.
In this case: testRowFilterBinaryComparatorEquals